hivemind_core/database.py

```
@@ -1,254 +1,716 @@
+        Compares two Client instances for equality based on their serialized data.
+
+        Args:
+            other: The other Client or Client-compatible object to compare with.
+
+        Returns:
+            True if the clients are equal, False otherwise.
+        """
+        try:
+            other = cast2client(other)
+        except:
+            pass
```

Ruff (0.8.2)

Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

Replace with `contextlib.suppress(Exception)`

(SIM105)